src/filters/oidc/jwks_resolver.cc

```diff
@@ -37,8 +42,22 @@ void DynamicJwksResolverImpl::JwksFetcher::request(
       }
     }
 
+    constexpr auto default_interval_with_no_jwks = std::chrono::seconds(3);
```

Comment:
following envoy code practice, if I remember correctly, constants are usually defined outside the function, starting with letter `kXXX`.

Reply:
also feel the var can be named a bit different for readability.

```cpp
// The number of seconds dynamic jwks resolver will retry to request JWKS during the startup time.
// A short interval ensures us can get JWKS ready before serving the traffic.
constexpr int kRefreshSecondsBeforeInitialized = 3;
```